melodic/distribution.yaml

```diff
@@ -4246,19 +4246,26 @@ repositories:
       version: releasePackage
     release:
       packages:
+      - collision_avoidance
+      - control_bringup
+      - gcs_interface
+      - gimbal
+      - gnss_utils
+      - gpu_voxels_ros
+      - gstreamer
       - heifu
-      - heifu_bringup
-      - heifu_description
-      - heifu_diagnostic
-      - heifu_mavros
-      - heifu_msgs
-      - heifu_safety
-      - heifu_simple_waypoint
-      - heifu_tools
+      - heifu-bringup
```

Comment:
These packages appear to have changed names to something that does not match our naming convention. https://www.ros.org/reps/rep-0144.html